server/chat-commands.js

```diff
@@ -621,20 +621,29 @@ const commands = {
 	'!away': true,
 	idle: 'away',
 	afk: 'away',
-	away(target, room, user) {
-		this.parse('/blockchallenges');
-		this.parse(`/blockpms ${target}`);
+	brb: 'away',
+	away(target, room, user, connection, cmd) {
+		let awayMessage = toId(cmd);
+		if (target) {
+			awayMessage = Chat.namefilter(target, user);
+			if (!awayMessage) return;
+		} else {
+			if (awayMessage === 'afk' || awayMessage === 'brb') awayMessage = awayMessage.toUpperCase();
+			awayMessage = `${awayMessage[0].toUpperCase()}${awayMessage.slice(1)}`;
```

Comment:
Place this in an `else` or a ternary statement. Sure, recapitalizing the first letter of AFK or BRB is a noop, but its needless and makes the code less clear.